guides/common/modules/ref_email-notification-types.adoc

```diff
@@ -3,14 +3,26 @@
 
 {Project} can create the following email notifications:
 
-* *Audit summary*: A summary of all activity audited by {ProjectServer}.
-* *Host built*: A notification sent when a host is built.
-* *Host errata advisory*: A summary of applicable and installable errata for hosts managed by the user.
-* *Compliance policy summary*: A summary of OpenSCAP policy reports and their results.
-* *Promote errata*: A notification sent only after a Content View promotion.
+Audit summary:: A summary of all activity audited by {ProjectServer}.
+ifdef::satellite[]
+{SmartProxy} sync failure:: A notification sent after {SmartProxy} synchronization fails.
+endif::[]
+Compliance policy summary:: A summary of OpenSCAP policy reports and their results.
+Content view promote failure:: A notification sent after content view promotion fails.
+Content view publish failure:: A notification sent after content view publication fails.
+Host built:: A notification sent when a host is built.
```

Comment:
"is built" or "has been built"? Or maybe "after" instead of "when"?

Reply:
Let's go with "after". Addressed.